pkg/sql/distsqlrun/processors.proto

```diff
@@ -353,6 +353,11 @@ message BackfillerSpec {
   // along with a backfill for the rows. This is the maximum number
   // of entries backfilled per chunk.
   optional int64 chunk_size = 5 [(gogoproto.nullable) = false];
+
+  // other_tables contains any other (leased) table descriptors necessary for the
+  // backfiller to do its job, such as the descriptors for tables with fk
+  // relationships to the table being modified.
+  repeated sqlbase.TableDescriptor other_tables = 6 [(gogoproto.nullable) = false];
```

Comment:
As discussed foreign key constraints can be checked before and after column backfill and so need not be passed to each processor.